README.md

```diff
@@ -15,102 +15,126 @@ This project demonstrates how to deploy a volume based on NFSv4.1 protocol with
+1. This project is built upon Maven, which must be installed in order to run the sample. Instructions on installing Maven can be found on their website [here](https://maven.apache.org/install.html)
+1. Azure subscription
+1. Subscription needs to be whitelisted for Azure NetApp Files. For more information, see
+   [Submit a waitlist request for accessing the service](https://docs.microsoft.com/azure/azure-netapp-files/azure-netapp-files-register#waitlist).
+1. Resource Group created
+1. Virtual Network with a delegated subnet to Microsoft.Netapp/volumes resource. For more information, see
+   [Guidelines for Azure NetApp Files network planning](https://docs.microsoft.com/en-us/azure/azure-netapp-files/azure-netapp-files-network-topologies).
+1. For this sample console application to work, authentication is needed. We will use Service Principal based authentication
+    1. Within an [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/quickstart) session, make sure
+       you're logged in at the subscription where you want to be associated with the service principal by default:
         ```bash
         az account show
+       ```
+       If this is not the correct subscription, use:
+         ```bash
+        az account set -s <subscription name or id>  
         ```
-        If this is not the correct subscription, use             
-          ```bash
-         az account set -s <subscription name or id>  
-         ```
-    1. Create a service principal using Azure CLI
+    1. Create a service principal using Azure CLI:
         ```bash
         az ad sp create-for-rbac --sdk-auth
         ```
 
-        >Note: this command will automatically assign RBAC contributor role to the service principal at subscription level, you can narrow down the scope to the specific resource group where your tests will create the resources.
+       >Note: This command will automatically assign RBAC contributor role to the service principal at subscription level.
+       You can narrow down the scope to the specific resource group where your tests will create the resources.
+
+    1. Set the following environment variables from the output of the creation:
 
-    1. Copy the output content and paste it in a file called azureauth.json, secure it with file system permissions and save it outside the tree related of your 	local git repo folder so the file doesn't get commited. 
-    1. Set an environment variable pointing to the file path you just created, here is an example with Powershell and bash:
-        Powershell 
+       Powershell
         ```powershell
-       [Environment]::SetEnvironmentVariable("AZURE_AUTH_LOCATION", "C:\sdksample\azureauth.json", "User")
-       ```
-        Bash
+        $env:AZURE_SUBSCRIPTION_ID = <subscriptionId>
```

Comment:
why are we changing the way we are reading the variables ? I mean to read one file instead of assigning them individually